DependencyInjection/PetitPressGpsMessengerExtension.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PetitPress\GpsMessengerBundle\DependencyInjection;
+
+use PetitPress\GpsMessengerBundle\Transport\GpsTransportFactory;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\PhpFileLoader;
+use Symfony\Component\DependencyInjection\Reference;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+
+final class PetitPressGpsMessengerExtension extends Extension
```

Comment:
Changing the name of the extension is a BC break.
v1.4 break tests in a package I work on.